docs/decisions/0001-course-experience.rst

```diff
@@ -0,0 +1,283 @@
+Courses units in LMS
+--------------
+
+Status
+======
+
+Proposed
+
+Context
+=======
+
+NELP business case requires multiples features that are not part
+of open-edx platform core, and it's necessary a new implementation
+that allows users to report and share their opinions about the
+course content and experience during the course.
+
+Decisions
+=========
+
+1. Create a course experience module.
+2. Create ``RateCourse`` model with the following fields.
+
+  * author => Foreign key to user model, the user who set their opinion.
+
+  * status => String field with choices: Liked, Unliked and not-set.
```

Comment:
here i have some interesting discussion about if it would be a string field. Mainly it could be  lot of data consumption saving in storage `liked` for each user. this could be a boolean or (1,0). Also the not-set could be Null.

thinking in that the not-liked has a very business discussion because will the teacher see the dislikes xD. for example in instagrama everything is like xd

Reply:
I think a boolean is a good option, the content that the instructor or student will see is out of the scope of this proposal